app/src/main/kotlin/org/eclipse/kuksa/companion/application/ApplicationViewModel.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2023-2024 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.eclipse.kuksa.companion.application
+
+import androidx.lifecycle.ViewModel
+import org.eclipse.kuksa.DataBrokerConnection
+import org.eclipse.kuksa.DisconnectListener
+import kotlin.properties.Delegates
+
+class ApplicationViewModel : ViewModel() {
```

Comment:
I would rather see these vars inside the ConnectionViewModel because there is not really a view linked to the Application level. You can leave the application package if you want. Will probably be used later with hilt anyway. :)